docs/design-tech/redis-schema.md

```diff
@@ -33,6 +33,15 @@ Values are json encoded.
   + `x` is the `uuid` of the file.
   + size (value) is a json encoded `FileUsage` object 
 
+# Stripe Billing
+
++ `public_key:x:stripe_user_info`
+  + `x` is the `public_key`.
+  + Basic stripe information for that user.
++ `public_key:x:in_stripe_billing_workflow`.
```

Comment:
I bet this is doable with one key, let me keep reviewing to see.

Reply:
You're missing one of your keys, you're missing the reverse lookup from customer to public key.